client/__tests__/util/annoMatrix/annoMatrix.test.ts

```diff
@@ -34,15 +35,15 @@ describe("AnnoMatrix", () => {
       expect(annoMatrix.nObs).toEqual(serverMocks.schema.schema.dataframe.nObs);
       expect(annoMatrix.nVar).toEqual(serverMocks.schema.schema.dataframe.nVar);
       expect(annoMatrix.isView).toBeFalsy();
-      expect(annoMatrix.viewOf).toBeUndefined();
```

Comment:
@bkmartinjr this looks like the opposite of the original assertion (from undefined to now annomatrix), does this indicate runtime code change or something else? Thank you!

Reply:
@tihuan - please review Mim's comments and discussion for an explanation.  The runtime changed.

Reply:
@tihuan, discussion is [here](https://github.com/chanzuckerberg/cellxgene/pull/2371#discussion_r687314871).

Reply:
Oh perfect! Thanks so much, both 🙏